source/blog/2018-10-08-ember-3-5-released.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Ember 3.5 Released
+author: Melanie Sumner, Jen Weber
+tags: Releases, 2018, 3, 3.5
+responsive: true
+---
+
+Today the Ember project is releasing version 3.5 of Ember.js, Ember Data, and Ember CLI. This release kicks off the 3.5 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+Additionally, 3.4 is now promoted to LTS, which stands for Long
+Term Support. An LTS version of Ember continues to receive security updates
+for 9 release cycles (54 weeks)
+and bugfixes for 6 cycles (36 weeks).
+LTS releases typically occur every four minor versions, plus the
+first and last releases of major versions. 
+So far, the LTS releases for 3.x are 3.0 and 3.4.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.5
+Ember.js 3.5 is an incremental, backwards compatible release of Ember with bugfixes. 
+It is common for minor releases to provide bugfixes that pave the way for
+release of new features in future releases.
+There are zero (0) new features, zero (0) deprecations, and five (5) minor 
```

Comment:
It's (2) bugfixes because the rest were all backported to 3.4.x

Reply:
Edited.